docs/user-guide/options.md

```diff
@@ -40,12 +40,12 @@ For CSS with standard syntax, Stylelint uses [postcss-safe-parser](https://www.n
 
 When using the Node.js API, the autofixed code is available as the value of the `code` property in the returned object.
 
-If a source contains a:
+---
 
-- scoped disable comment, e.g. `/* stylelint-disable color-named */`, any problems reported by the scoped rules will not be automatically fixed anywhere in the source
-- unscoped disable comment, i.e. `/* stylelint-disable */`, the entirety of source will not be automatically fixed
+When a rule relies on the deprecated `context.fix` and a source contains:
```

Comment:
[suggest] Minor. Making `context.fix` a link may be nice for people who want to know the deprecation. For example:

```suggestion
When a rule relies on the deprecated [`context.fix`](../developer-guide/rules.md#context) and a source contains:
```

Reply:
```suggestion
When a rule relies on the deprecated [`context`](../developer-guide/rules.md#context).fix` and a source contains:
```